ChangeLog

```diff
@@ -30,6 +30,7 @@ ver. 0.9.5 (2016/XX/XXX) - wanna-be-released
    * New Actions:
      - action.d/firewallcmd-rich-rules and action.d/firewallcmd-rich-logging (gh-1367)
 - Enhancements:
+   * extremely speedup of all sqlite database operations (gh-1436)
```

Comment:
Extreme ... by using temp_store memory

Also while at it might be worth unifying leading casing

Shouldn't it be an option in fail2ban.conf so if it doesn't work for them they could quickly disable it?